### DISCUSSION TITLE
Execute Multiple Operation in WASM.

### DISCUSSION THREAD

**Discussion by akash-goel** [duckdb/duckdb#6069] [General]
Hi Team,

we want to run multiple operation like Casting , Adding new column , Filters on parquet dynamically . we want to perform the operation in WASM. Below Option we thought

1- Creating Subqueries to perform multiple operations and get the final data.
2- Create the DuckDB table  and persist the data in table , execute the operations and than get the final data.

i would like to check is there any way to execute multiple operation without using subqueries and persisting the data in table ?

Regards,
Akash

**Comment by carlopi** [comment]
Hi @akash-goel, currently this is not possible, but you are not the first person to suggest this, see:
- https://github.com/duckdb/duckdb/discussions/5629
- https://github.com/duckdb/duckdb-wasm/discussions/1131#discussioncomment-4782573
(and potentially other that I am missing)

Basic answer is "no, not currently possible", but feel free to continue the discussion in https://github.com/duckdb/duckdb/discussions/5629, so we have a single place where the conversation about this feature proposal goes forward?

Thanks!
Carlo
